recipes-thirdparty/dab-adapter/files/0005-PREMIUMAPP-3142-Remove-AmazonAlexa-from-the-list-of-.patch

```diff
@@ -0,0 +1,150 @@
+From c073ede71b1567ab38a254863f9c1421e260cff0 Mon Sep 17 00:00:00 2001
+From: Adam Stolcenburg <adam_stolcenburg@comcast.com>
+Date: Tue, 14 Jan 2025 12:29:56 +0100
+Subject: [PATCH 5/5] PREMIUMAPP-3142 Remove AmazonAlexa from the list of
+ supported voice systems
+
+---
+ src/device/rdk/voice/list.rs       | 4 ++++
+ src/device/rdk/voice/send_audio.rs | 7 +++++++
+ src/device/rdk/voice/send_text.rs  | 8 ++++++++
+ src/device/rdk/voice/set.rs        | 8 ++++++++
+ 4 files changed, 27 insertions(+)
+
+diff --git a/src/device/rdk/voice/list.rs b/src/device/rdk/voice/list.rs
+index 1b5de47..ed73e96 100644
+--- a/src/device/rdk/voice/list.rs
++++ b/src/device/rdk/voice/list.rs
+@@ -1,9 +1,11 @@
+ use crate::dab::structs::DabError;
+ use crate::dab::structs::ListVoiceSystemsResponse;
+ use crate::dab::structs::VoiceListRequest;
++/*
```

Comment:
This patch is disabling complete feature for Voice related RDK plugin abstraction and this is not the right approach in my opinion.
The proper way to do this is:
- Make sure that the platform API is not claiming AVS support when it does not. Ref: In https://rdkcentral.github.io/rdkservices/#/api/VoiceControlPlugin?id=voicestatus urlPtt shall not have `avs://` end point. This need to be handled at respective RDK component integration level.
- When the platform does not say it supports voice, the DAB test suite is not supposed to invoke related test cases.